python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
Why is a user needing to copy their files to `tmpDir` and why could they not have simply set `localDir=tmpDir` in the constructor when they created their oututfiles list?